pages/common/wapm.md

```diff
@@ -0,0 +1,36 @@
+# wapm
+
+> The WebAssembly package manager.
+> More information: <https://wapm.io/help/reference>.
+
+- Interactively create a wapm.toml file:
```

Comment:
Perhaps this will read better?
```suggestion
- Interactively create a new `wapm.toml` file:
```

Not entirely sure on  the backticks. Thoughts?

Reply:
We don't use them in general.